src/app-host/live-reload-client.js

```diff
@@ -78,13 +65,15 @@ module.exports.start = function (sock) {
     }
 
     /**
-     * Finds all the DOM elements that have a reference attribute ("url", "href" or "src") pointing to the given relative path. Excludes <script> tags.
+     * Finds all the DOM elements that have a reference attribute ("url", "href" or "src") pointing to the given
+     * relative path. Excludes <script> tags.
      *
      * @param {String} fileRelativePath The URL of the file to check, relative to the webRoot.
      * @returns {{ domNode: Element, referenceAttribute: string }[]} An array of "resources" referencing the given file.
      */
     function findDomNodesForFilePath(fileRelativePath) {
-        // To use querySelectorAll to query elements based on their attributes, the selector's syntax is: '[attrib1], [attrib2], ...'.
+        // To use querySelectorAll to query elements based on their attributes, the selector's syntax is:
+        // '[attrib1], [attrib2], ...'.
         var selectorString = `[${referenceAttributes.join('], [')}]`;
```

Comment:
Templates is a feature of ecmascript 6, I know that we are targeting like new browsers, but maybe we can consider using features of the language and APIs that are most common in other browsers too until we have full support of it. I don't know the state of the support of templates, so this is just a comment :)